hugegraph-core/src/main/java/com/baidu/hugegraph/backend/query/Query.java

```diff
@@ -88,6 +92,8 @@ public Query(HugeType resultType, Query originQuery) {
 
         this.aggregate = null;
         this.showExpired = false;
+        this.olap = false;
+        this.olapPks = new IdSet(CollectionType.EC);
```

Comment:
also replace conditions and ids to EC collection

Reply:
ids need keep order, using LinkedHashSet, can't be replaced